elements/lisk-transactions/src/fee.ts

```diff
@@ -37,14 +38,18 @@ const computeTransactionMinFee = (
 	trx: Record<string, unknown>,
 	options?: Options,
 ): bigint => {
+	const mockSignatures = new Array(
```

Comment:
Is it mock? maybe name it `emptySignatures`?

Reply:
it's both 64bytes and 0 bytes, so empty is not correct I think

Reply:
dummySignatures?